.circleci/config.yml

```diff
@@ -342,21 +342,21 @@ workflows:
           filters:
             branches:
               only: master
-      - ecr-build-and-push-image-development:
-          requires:
-            - terraform-init-and-apply-to-development
-          filters:
-            branches:
-              only: master
-      - terraform-deploy-ecs-task-to-development:
-          requires:
-            - ecr-build-and-push-image-development
-          filters:
-            branches:
-              only: master
+      # - ecr-build-and-push-image-development:
+      #     requires:
+      #       - terraform-init-and-apply-to-development
+      #     filters:
+      #       branches:
+      #         only: master
+      # - terraform-deploy-ecs-task-to-development:
+      #     requires:
+      #       - ecr-build-and-push-image-development
+      #     filters:
+      #       branches:
+      #         only: master
       - deploy-to-development:
           requires:
-            - terraform-deploy-ecs-task-to-development
+            - terraform-init-and-apply-to-development:
```

Comment:
nearly there - you hooked deploy job back onto requires chain, but you now need to remove the colon - it's a syntax thing.